modules/ossm-installing-multi-cluster-mesh.adoc

```diff
@@ -0,0 +1,179 @@
+:_mod-docs-content-type: Procedure
+[id="ossm-installing-multi-cluster-mesh"]
+= Installing a multi-cluster mesh 
+:context: ossm-installing-multi-cluster-mesh
+
+This procedure explains how to install the {SMProduct} 3.0 Operator and the Sail CRDs to manage a multi-cluster {istio} deployment.
+
+.Prerequisites
+
+* You have installed the {SMProduct} 3.0 Operator and the Sail Custom Resource Definitions (CRDs) to every cluster.
+
+* You have installed `istioctl`.
+
+* You have access to two {ocp-product-title} clusters with external lb support. 
+
+* You have created a `kubeconfig` file with a context for each cluster.
+
+.Procedure
+
+. Set up the environment variables:
++
+[source,terminal]
+----
+$ export CTX_CLUSTER1=<cluster1-ctx>
+$ export CTX_CLUSTER2=<cluster2-ctx>
+$ export ISTIO_VERSION=1.23.2
+----
+
+. Create a namespace called `istio-system` for the first cluster by running the following command at the terminal:
++
+[source,terminal]
+----
+$ oc get ns istio-system --context "${CTX_CLUSTER1}" || kubectl create namespace istio-system --context "${CTX_CLUSTER1}"
+----
+
+. Create a namespace called `istio-system`  for the second cluster:
++
+[source,terminal]
+----
+$ oc get ns istio-system --context "${CTX_CLUSTER2}" || kubectl create namespace istio-system --context "${CTX_CLUSTER2}"
+----
+
+. Create a shared root certificate:
++
+[source,terminal]
+----
+$ openssl genrsa -out root-key.pem 4096
+encrypt_key = no
+prompt = no
+utf8 = yes
+default_md = sha256
+default_bits = 4096
+req_extensions = req_ext
+x509_extensions = req_ext
+distinguished_name = req_dn
+[ req_ext ]
+subjectKeyIdentifier = hash
+basicConstraints = critical, CA:true
+keyUsage = critical, digitalSignature, nonRepudiation, keyEncipherment, keyCertSign
+[ req_dn ]
+O = Istio
+CN = Root CA
+
+openssl req -sha256 -new -key root-key.pem \
+  -config root-ca.conf \
+  -out root-cert.csr
+
+openssl x509 -req -sha256 -days 3650 \
+  -signkey root-key.pem \
+  -extensions req_ext -extfile root-ca.conf \
+  -in root-cert.csr \
+  -out root-cert.pem
+----
+
+. Create an intermediate certificate for the east cluster:
++
+[source,terminal]
+----
+$ openssl genrsa -out east/ca-key.pem 4096
+$east/intermediate.conf
+[ req ]
+encrypt_key = no
+prompt = no
+utf8 = yes
+default_md = sha256
+default_bits = 4096
+req_extensions = req_ext
+x509_extensions = req_ext
+distinguished_name = req_dn
+[ req_ext ]
+subjectKeyIdentifier = hash
+basicConstraints = critical, CA:true, pathlen:0
+keyUsage = critical, digitalSignature, nonRepudiation, keyEncipherment, keyCertSign
+subjectAltName=@san
+[ san ]
+DNS.1 = istiod.istio-system.svc
+[ req_dn ]
+O = Istio
+CN = Intermediate CA
+L = $cluster
```

Comment:
$cluster needs to be replaced with `east`

Reply:
done